modules/new_serverpod_auth/serverpod_auth_email_account/serverpod_auth_email_account_server/test/integration/email_accounts_admin_test.dart

```diff
@@ -11,14 +11,15 @@ import 'test_tools/serverpod_test_tools.dart';
 void main() {
```

Comment:
@coderabbitai check this test file, ensuring that all newly added methods from this PR are properly tested.

Double-check that test descriptions match their implementation.

Check the spelling.